packages/Pager/src/pager.scss

```
@@ -1,192 +1,292 @@
+    --ids-pager-height: #{tokens.$space-5};
+    --ids-pager-line-height: #{tokens.$line-height-xxxl};
+
+    /* Item */
+    --ids-pager-item-wrapper-border-size: 0;
+    --ids-pager-item-wrapper-padding: 0;
+    --ids-pager-item-wrapper-radius: 0;
+    --ids-pager-item-padding: #{tokens.$space-1};
+    --ids-pager-item-background: #{tokens.$samoyed};
+    --ids-pager-item-border: .1rem solid #{tokens.$grey-300};
+    --ids-pager-item-prev-border-radius: #{tokens.$border-radius-sm} 0 0 #{tokens.$border-radius-sm};
+    --ids-pager-item-next-border-radius: 0 #{tokens.$border-radius-sm} #{tokens.$border-radius-sm} 0;
+    --ids-pager-item-color: #{tokens.$grey-800};
+    --ids-pager-item-radius: 0;
+    --ids-pager-item-min-width: 3.2rem;
+    --ids-pager-item-z-index: #{tokens.$z-index-10};
+    --ids-pager-item-vertical-offset: -.1rem;
+    --ids-pager-item-prev-next-offset: -.1rem;
+
+    /* Selected Item */
+    --ids-pager-item-selected-background: #{tokens.$electric-blue-500};
+    --ids-pager-item-selected-color: #{tokens.$samoyed};
+
+    /* Hover */
+    --ids-pager-item-hover-border: .1rem solid #{tokens.$grey-600};
+    --ids-pager-item-hover-background: transparent;
+
+    /* Focus */
+    --ids-pager-item-focus-border-width: .1rem;
+    --ids-pager-item-focus-outline-width: 0;
+    --ids-pager-item-focus-color: #{tokens.$electric-blue-600};
+
+    /* Disabled */
+    --ids-pager-item-disabled-background: #{tokens.$grey-300};
+    --ids-pager-item-disabled-border: .1rem solid #{tokens.$grey-300};
+    --ids-pager-item-disabled-color: #{tokens.$grey-600};
+
+    /* Arrows */
+    --ids-pager-arrow-color: #{tokens.$grey-500};
+
+    /* Ellipsis */
+    --ids-pager-ellipsis-border: .1rem solid transparent;
+    --ids-pager-ellipsis-color: #{tokens.$grey-600};
+    --ids-pager-ellipsis-offset: 0;
+
+    /* Results */
+    --ids-pager-results-color: #{tokens.$grey-600};
+    --ids-pager-results-margin: 0 0 0 #{tokens.$space-3};
+}
 
-  /* Results */
-  --ids-pager-results-color: #{tokens.$grey-600};
-  --ids-pager-results-margin: 0 0 0 #{tokens.$space-3};
+[data-brand="workleap"] {
+    /* Default */
+    --ids-pager-font-family: var(--hop-body-sm-font-family);
+    --ids-pager-font-size: var(--hop-body-sm-font-size);
+    --ids-pager-height: 2rem;
+    --ids-pager-line-height: var(--hop-body-sm-line-height);
+
+    /* Item */
+    --ids-pager-item-wrapper-border-size: 0.0625rem;
+    --ids-pager-item-wrapper-padding: calc(var(--hop-space-inset-xs) - var(--ids-pager-item-wrapper-border-size));
+    --ids-pager-item-wrapper-radius: var(--hop-shape-rounded-md);
+    --ids-pager-item-padding: 0;
+    --ids-pager-item-background: var(--hop-neutral-surface);
+    --ids-pager-item-border: none;
+    --ids-pager-item-prev-border-radius: var(--hop-shape-rounded-md);
+    --ids-pager-item-next-border-radius: var(--hop-shape-rounded-md);
+    --ids-pager-item-color: var(--hop-neutral-text);
+    --ids-pager-item-radius: 999px;
```

Use the hopper tokens 
```suggestion
    --ids-pager-item-radius: var(--hop-shape-circle);
```